docs/mappings/flinkster_gbfs_2.3_mapping.md

```diff
@@ -0,0 +1,206 @@
+
+# Flinkster to GBFS Mapping
+
+This document maps the Flinkster API to GBFS.
+
+# Reference version
+
+This documentation refers to **[v2.3](https://github.com/MobilityData/gbfs/blob/v2.3/gbfs.md)**.
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [General Information](#general-information)
+* [Open Issues or Questions](#open-issues-or-questions)
+* [Files](#files)
+    * [gbfs.json](#gbfsjson)
+    * [gbfs_versions.json](#gbfs_versionsjson) *(added in v1.1)*
+    * [system_information.json](#system_informationjson)
+    * [vehicle_types.json](#vehicle_typesjson) *(added in v2.1)*
+    * [station_information.json](#station_informationjson)
+    * [station_status.json](#station_statusjson)
+    * [free_bike_status.json](#free_bike_statusjson)
+    * [system_hours.json](#system_hoursjson)
+    * [system_calendar.json](#system_calendarjson)
+    * [system_regions.json](#system_regionsjson)
+    * [system_pricing_plans.json](#system_pricing_plansjson)
+    * [system_alerts.json](#system_alertsjson)
+    * [geofencing_zones.json](#geofencing_zonesjson) *(added in v2.1)*
+
+## Introduction
+
+This specification describes the mapping of car-sharing provider [Flinkster](https://www.flinkster.de/)'s API to GBFS.
+
+
+## General Information
+
+This section gives a high level overview of the sharing provider's API and the defined mapping to GBFS.
+
+The Deutsche Bahn Connect Customer B2B API is available via a [API documentation](https://dbconnect-b2b-prod.service.dbrent.net/customer-b2b-api/docs/customer-b2b-api.html).
+
+
+## Open Issues or Questions
+
+Open questions are managed as [issues](https://github.com/mobidata-bw/x2gbfs/issues?q=is%3Aissue+is%3Aopen+label%3Aflinkster). They should be tagged with `flinkster`.
+
+
+## Files
+
+### gbfs.json
+
+Standard file, feed will be provided in feed language `de` only.
+
+### gbfs_versions.json
+
+Optional file, will not be provided. Only GBFSv2.3 supported.
+
+### system_information.json
+
+System information is manually extracted from the providers homepage. Only the real configuration in [config/flinkster.json](config/flinkster.json) is relevant.
```

Comment:
The config/flinkster.json link is not accessible.

Reply:
Fixed with a648fc4c7aae563c5769bfd386c446014d8cb057